test/Sentry.Serilog.Tests/IntegrationTests.LoggingInsideTheContextOfLogging.verified.txt

```diff
@@ -0,0 +1,41 @@
+{
+  diagnosticLoggerEntries: [
+    {
+      Level: error,
+      Message: Reentrant log event detected. Logging when inside the scope of another log event can cause a StackOverflowException. LogEventInfo.Message: message from OnEvaluating
+    }
+  ],
+  Envelopes: [
+    {
+      Header: {
+        event_id: Guid_1,
+        sdk: {
+          name: sentry.dotnet,
```

Comment:
Shouldn't these be coming through as `sentry.dotnet.serilog` ?

Reply:
the root header is still `sentry.dotnet`. currently `sentry.dotnet.serilog` is in the envelope, but is being scrubbed. i will do a diff PR to do better at scrubbing.